connections.py

```diff
@@ -0,0 +1,28 @@
+from pyspark.sql import SparkSession
+
+spark = SparkSession.builder.appName('MyProject') \
+    .master('local[*]') \
+    .getOrCreate()
+
+
+def read_tsv(file_name):
+    '''
+    Read data frame from tsv file
+    '''
+    dataset = spark.read.load(('dataset/' + file_name + '/data.tsv'),
+                              format='csv',
```

Comment:
`.csv` are you sure?

Reply:
Yes. because if this is removed or changed to a different value, then the inferschema throws an exception